Snippets/ASB/ASB_8/MultipleNamespaces.cs

```diff
@@ -80,6 +80,24 @@ void NamespaceRoutingRegistration(EndpointConfiguration endpointConfiguration)
         #endregion
     }
 
+    void NamespaceRoutingEndpointRegistration(EndpointConfiguration endpointConfiguration)
+    {
+        #region namespace_routing_endpoint_registration
+
+        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
+
+        var routing = transport.Routing();
```

Comment:
I'd add comments with steps. Something among the lines of 
// Step 1 - associate MyMessage with "sales" endpoint
...
// Step 2 - register namespace that will be used for "sales" endpoint
...
// Step 3 - Associated endpoint "sales" with the namespace to route MyMessage messages to